modules/logging-config-using-crs.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * logging/logging-configuration.adoc
+
+:_content-type: CONCEPT
+[id="logging-config-crs_{context}"]
+= Using Custom Resources
```

Comment:
Seems odd to  discuss such a general topic in the logging docs. You can't do anything in openshift without knowing what custom resources are.